src/test/java/NotificationMessageTest.java

```diff
@@ -0,0 +1,18 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class NotificationMessageTest extends BaseTest {
+
+    @Test
+    public void notificationMessageTest() {
+        driver.get(Constants.ALERTS_URL);
+        WebElement elementClickHere = driver.findElement(By.linkText("Click here"));
+        elementClickHere.click();
+        WebElement elementAlertText = driver.findElement(By.id("flash"));
+        String alertText = elementAlertText.getText();
+        String expectedText = "Action successful";
+        Assert.assertTrue(alertText.contains(expectedText));
```

Comment:
This test will fail even if 'Action successful' appeared, please fix it
![Screenshot_15](https://user-images.githubusercontent.com/13552476/154921140-dca4a983-a5e9-49bc-8ce7-2c31288e1a2d.png)

Reply:
but according to screenshot we verify if alertText.contains(expectedText) is true. And on screenshot it is actually true. In other words here we verify if true==true. Seems it should pass. Or am I missing something?

Reply:
Yes, you're right, there is contains method